somnio_starter/__brick__/{{project_name.snakeCase()}}/packages/user_repository/lib/src/user_repository.dart

```diff
@@ -2,6 +2,7 @@ import 'dart:async';
 
 import 'package:api_client/api_client.dart';
 import 'package:auth_client/auth_client.dart';
+import 'package:flutter/foundation.dart';
```

Comment:
Is this necessary?

Reply:
Yes, it's for the visibleForTesting notation